cdk/package.json

```diff
@@ -2,34 +2,71 @@
   "name": "cdk",
   "version": "0.0.0",
   "private": true,
-  "bin": {
-    "cdk": "bin/cdk.js"
-  },
   "scripts": {
     "build": "tsc",
-    "watch": "tsc -w",
     "test": "jest",
-    "test:dev": "jest --watch",
+    "test-update": "jest -u",
     "format": "prettier --write \"{lib,bin}/**/*.ts\"",
-    "cdk": "cdk",
     "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
     "synth": "cdk synth --path-metadata false --version-reporting false",
     "diff": "cdk diff --path-metadata false --version-reporting false"
   },
   "devDependencies": {
-    "@guardian/eslint-config-typescript": "10.0.1",
-    "@types/jest": "^27.0.2",
-    "@types/node": "20.14.5",
-    "aws-cdk": "2.136.1",
-    "aws-cdk-lib": "2.136.1",
+    "@guardian/cdk": "58.1.2",
+    "@guardian/eslint-config-typescript": "8.0.0",
```

Comment:
Looks like we're downgrading this one as it was manually upgraded yesterday - https://github.com/guardian/elastic-search-monitor/pull/125. It might be worth sticking with the latest version if we can?

Reply:
Ah, I missed that. Will restore.

Reply:
The version of `eslint` on this branch is later than on #125, so we only need to update `@guardian/eslint-config-typescript` (and `typescript`) - https://github.com/guardian/elastic-search-monitor/pull/127/commits/ae76058bd362fc112828f24fa97715a2f3192f98.